jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/JfrMetrics.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jfr.metrics;
+
+import io.opentelemetry.api.metrics.MeterProvider;
+import io.opentelemetry.contrib.jfr.metrics.internal.RecordedEventHandler;
+import java.util.concurrent.Executors;
+import java.util.function.Consumer;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import jdk.jfr.EventSettings;
+import jdk.jfr.consumer.RecordingStream;
+
+public final class JfrMetrics {
+  private JfrMetrics() {}
+
+  private static final Logger logger = Logger.getLogger(JfrMetrics.class.getName());
+
+  public static void enableJfr(MeterProvider meterProvider) {
```

Comment:
```suggestion
  public static void enable(MeterProvider meterProvider) {
```

Class has Jfr in it don't think method needs it